plugins/VisitFrequency/Reports/Get.php

```diff
@@ -32,14 +32,25 @@ protected function init()
         $this->processedMetrics = array(
             new ReturningMetric(new AverageTimeOnSite()),
             new ReturningMetric(new ActionsPerVisit()),
-            new ReturningMetric(new BounceRate())
+            new ReturningMetric(new BounceRate()),
```

Comment:
just for readability be good to add `_returning` for the first three returning metrics as a second parameter too then it is more clear. Also I see we use these suffix a few times in the code maybe we could use a constant for it? Also in the `Archive` class where we detect plugin etc (see first change in this diff on github).